### PR TITLE
Integrate styled-component in Custom Theme Changer, Add Mobile View

### LINKED ISSUE #2412: Integrate styled-component in Custom Theme Changer, Add Mobile View
**Actual Behaviour**

Mobile view is broken for theme changer.

**Expected Behaviour**

Integrate styled-component in Custom Theme Changer, Add Mobile View

**Would you like to work on the issue?**

Yes

### PR DESCRIPTION
**Pull request by fragm3**
Fixes #2412

Changes: 
- [x] Integrate styled-component in Custom Theme Changer
- [x] Add Mobile View

Demo Link: https://pr-2415-fossasia-susi-web-chat.surge.sh

Screenshots of the change:

Desktop:

Screenshot 2019-07-02 at 10 41 48 PM

Mobile(414)

Screenshot 2019-07-02 at 10 15 26 PM

Screenshot 2019-07-02 at 10 15 30 PM

Mobile(375)

Screenshot 2019-07-02 at 10 15 37 PM

### CODE REVIEW

File: src/components/Settings/PreviewThemeChat.js [hunk 1]
```diff
@@ -80,6 +184,7 @@ class PreviewThemeChat extends Component {
   };
 
   render() {
+    const { messages, message } = this.state;
```
Comment (akshatnitd) [inline src/components/Settings/PreviewThemeChat.js]: Can you rename `message` to a more apt name. Currently, it is confusing.

**Comment by mariobehling** [comment]
Could you increase width and height?

Use more of the screen space, please.

Screenshot 2019-07-02 at 10 15 37 PM

**Comment by fragm3** [comment]
Fixed @mariobehling @akshatnitd 

Please refresh the page to see the changes in the mobile view. Also, the Dialog Actions are coming fine in mobile(not in chrome, due to larger height in mobile view in chrome).

Screenshot 2019-07-03 at 9 27 24 PM


Screenshot 2019-07-03 at 9 28 14 PM

**Comment by fragm3** [comment]
Resolved the conflicts @harsh-jindal, @samagragupta made the mobile view fixed for colorpicker

Screenshot 2019-07-04 at 7 53 33 PM

**Comment by fragm3** [comment]
@samagragupta which devices uses this these dimensions? Not of priority.

**Comment by fragm3** [comment]
@harsh-jindal This PR is specifically for mobile view. I don't think our aim here is to make the views responsive to every dimension. IMO, we need to focus just on Desktop, iPad and mobile view. @akshatnitd, thoughts?

**Comment by fragm3** [comment]
@akshatnitd It's due to higher height in chrome responsive view. On the mobile, it would look fine.

**Comment by harsh-jindal** [comment]
@fragm3 @akshatnitd Agreed the goal is to get it working on all devices, but it works well on almost all mobiles and laptops but not on tablets which are smaller than ipad and bigger than mobile view port. for eg. nexus 7. Also the other mobile view fixes were keeping in mind of these viewports too. So I guess it can be good practice to fix it for most viewports. We can leave it for this PR but maybe follow this in the future.

**Pull request merged**